core/include/scattered_array.h

```diff
@@ -83,7 +83,7 @@
  * @element_type: The type of the elemenet
  */
 #define SCATTERED_ARRAY_BEGIN(array_name, element_type) (__extension__({ \
-		static const element_type __scattered_array_begin[0] __unused \
+		static const element_type __scattered_array_begin[0] __aligned(32) __unused \
```

Comment:
Why do we still need __aligned(32) here?

Reply:
This place also needs X86_64 macro.

Reply:
This place also needs X86_64 macro.

Reply:
This place also needs X86_64 macro.

Reply:
#ifdef #else #endif used in a macro will cause build fail. Make a copy to add align in.